bin/before_install

```diff
@@ -18,3 +18,18 @@ elif [ ! -d "$spec_manageiq" ]; then
   echo "== Cloning manageiq sample app =="
   git clone https://github.com/ManageIQ/manageiq.git --branch oparin --depth 1 "$spec_manageiq"
 fi
+
+if [ -n "$CI" ]; then
+  echo "== Repository Owner: $GITHUB_REPOSITORY_OWNER =="
+  echo "== Base Ref: $GITHUB_BASE_REF =="
+  echo "== Ref Name: $GITHUB_REF_NAME =="
```

Comment:
I think we should remove these before merge.

Reply:
Forgot to mark this as WIP, this is only for testing the script since it will live in core but we can't cross-repo this yet